guide/english/java/data-abstraction/index.md

```diff
@@ -9,6 +9,16 @@ Likewise in Object-oriented programming, abstraction is a process of hiding the
 
 In Java, abstraction is achieved using Abstract classes and interfaces.
 
+Interfaces were introduced in Java to enhance Java’s single-inheritance model, multiple inheritance created too many problems for programmers and compiler writers. 
+All methods in interface are public, we may not write it every time, like all of them are abstract, but we never write the key word abstract.
```

Comment:
```suggestion
All methods in an Interface are implicitly `public` so we need not write the `public` keyword each time. 
```

Java 8 introduced default methods, which can contain full implementation, so "all of them are abstract" doesn't really hold true anymore.